seq2science/schemas/config/scrna.schema.yaml

```diff
@@ -29,5 +29,81 @@ properties:
 
   barcodefile:
     type: string
-
-  
+    
+  run_kb_seurat_pp:
+    default: false
+    type: boolean
+      
+  # Exploratory data analysis for in-house Celseq2
+  seurat:
+    description: Seurat rmd for exploratory data analysis
+    properties:
+      method: 
+        description: method -- experimental method
+        enum:  [384plate,droplet]
+      barcode_file:  
+        description: "barcode_file -- .tab delimited file with 2-col layout for plate based methods (with DNA barcode and well-id)"
```

Comment:
Is there a reason each description starts with the name of the key? I don't necessarily mind, but it is inconsistent with the rest of the schemas

Reply:
It comes straight from the markdown documentation to display the parameter name when knitting the document. I can remove it though.